cartoframes/utils/utils.py

```diff
@@ -272,11 +272,15 @@ def default(self, o):
         return super(CustomJSONEncoder, self).default(o)
 
 
-def encode_geodataframe(data):
+def parse_local_data(data, encode=True):
```

Comment:
I would prefer a more descriptive name :thinking:

Reply:
I agree! What about `enable_encoding`? Any ideas?

Reply:
Since we have `get_geodataframe_bounds`, something like `get_geodataframe_data` sounds good to me.